.github/keylabeler.yml

```diff
@@ -0,0 +1,49 @@
+# Determines if we search the title (optional). Defaults to true.
+matchTitle: true
+
+# Determines if we search the body (optional). Defaults to true.
+matchBody: true
+
+# Determines if label matching is case sensitive (optional). Defaults to true.
+caseSensitive: false
+
+# Explicit keyword mappings to labels. Form of match:label. Required.
+labelMappings:
+  "[WIP]": WIP
+  cicd: cicd
+  issue: issue
+  bug: BUG
+  error: BUG
+  help: help-wanted
+  guidance: help-wanted
+  feature: enhancement
+  enhancement: enhancement
+  feat: enhancement
+  docs: docs
+  Docs: docs
+  chore: skip-changelog
+  skip: skip-changelog
+  argocd: sak-argocd
+  argo-cd: sak-argocd
+  albcontroller: sak-alb
+  alb-controller: sak-alb
+  cognito: sak-cognito
+  Cognito: sak-cognito
```

Comment:
You used case-sensitive false. Why need both Cognito and cognito mapping? 
And can we using regexp to avoid double records for argocd and argo-cd and etc?

Reply:
You are right, I have removed dublicates and tested this.
![image](https://user-images.githubusercontent.com/42974336/134132964-2c559851-aa88-4961-a8bc-2e5a57bf96c0.png)